config/banners.yml

```diff
@@ -8,3 +8,9 @@ donate_2023:
     - [banners/donate_2023_1@2x.jpg, 2x]
   dark: true
   enddate: 2023-dec-31
+sotmeu_2023:
+  id: sotmeu_2023
+  alt: State of the Map Europe 2023
+  link: https://stateofthemap.eu/
+  img: StateoftheMapEurope_2023.png
```

Comment:
This needs to be `banners/StateoftheMapEurope_2023.png` for things to work - that's why the tests are failing.

Reply:
fixed